csrc/multidevice/communicator.h

```diff
@@ -86,9 +86,13 @@ class Communicator {
   }
 
   // returns the backend associated with a team
+  // the argument "prefix" is prepended to the key used to retrieve preexisting
+  // backends. This allow us to have a fine-grained control on how the backends
+  // are cached.
```

Comment:
nit: prefix is used to distinguish between different backends with the same team.